drivers/adc/adc_handlers.c

```diff
@@ -7,8 +7,21 @@
 #include <adc.h>
 #include <syscall_handler.h>
 
-_SYSCALL_HANDLER1_SIMPLE_VOID(adc_enable, K_OBJ_DRIVER_ADC, struct device *);
```

Comment:
I hate to see all these oneliners go away.
Consider _SYSCALL_DRIVER_HANDLER1_SIMPLE_VOID(api_name, op_name, api_struct, ...)
Not mandatory